Source/RxSwift/unwrap.swift

```diff
@@ -16,8 +16,8 @@ extension ObservableType {
 
      - returns: An observable sequence of non-optional elements
      */
-
-    public func unwrap<Result>() -> Observable<Result> where Element == Result? {
```

Comment:
why did you change the signature? It's best to keep the same as it was, just add the deprecation, nothing else.